coderd/database/migrations/000257_workspace_agent_script_timings.down.sql

```diff
@@ -0,0 +1,5 @@
+DROP TYPE IF EXISTS workspace_agent_script_timing_status CASCADE;
+DROP TYPE IF EXISTS workspace_agent_script_timing_stage CASCADE;
```

Comment:
note: `DROP TYPE ... CASCADE` is already done for `000246_provisioner_job_timings`